DTS-STN/secure-client-hub
# General clean up

## [ADO-133502](https://dev.azure.com/VP-BD/DECD/_workitems/edit/133502)

Fixed [AB#133502](https://dev.azure.com/VP-BD/d5ca3cd4-19cb-4c8c-b3e6-a7068f4677e4/_workitems/edit/133502)

### Changelog - "fix:" for bug fixes, "feat:" for features. Read more about Conventional Commits at https://www.conventionalcommits.org/en/v1.0.0/#summary

### Description of proposed changes:
This PR includes a number of assorted small changes including cleaning up stray comments, use of arbitrary values vs values defined in tailwind config and removal of files that aren't currently being used.

### What to test for/How to test
1. Pull in branch
2. Type `npm run dev`
3. Ensure app runs as expected
4. Type `npm run test:unit` and ensure all tests pass

### Additional Notes
From my understanding, the skeleton files were being kept around in the event that they "may be used in the future". I don't really like this approach as it just adds clutter. If we need these files in the future they will be available in git history.
